### PR TITLE
[Feature][86drr1f0a] Create the roi calculator thank you page

### PR DESCRIPTION
**Pull request by Mat0ias**
# Light It Pull Request Template ⚡⚡⚡

_Resolve_: [closes -86drr1f0a](https://app.clickup.com/t/86drr1f0a)

## Description and Requires ⭐

- [X] Implement the "thank you page" for the ROI calculator

## Proof of Test (optional) 📷

image

## Checklist ✅

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .vscode/settings.json
Comment (sgoycoechea-lightit) [inline .vscode/settings.json:1]: Is this change intentional?

File: apps/eo_web/src/router/routes.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ export const ROUTES = {
   profilingThankYou: "/profiling-thank-you",
   accountSetup: "/account-setup",
   roiCalculator: "/roi-calculator",
+  RoiCalculatorThankYou: "/roi-calculator-thank-you",
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/router/routes.tsx:25]:
```suggestion
  roiCalculatorThankYou: "/roi-calculator-thank-you",
```

File: packages/ui/tailwind.config.cjs [hunk 2]
```diff
@@ -1,4 +1,11 @@
 /** @type {import("tailwindcss").Config} */
 module.exports = {
   presets: [require("@eo/tailwind-config")],
+  theme: {
+    extend: {
+      fontFamily: {
+        "new-hero": ["New Hero", "sans-serif"],
+      },
+    },
+  },
```
Comment (sgoycoechea-lightit) [inline packages/ui/tailwind.config.cjs:10]: I don't think this is needed, we were using the new hero font already.

File: apps/eo_web/src/screens/RoiCalculatorThankYou.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+export const RoiCalculatorThankYou = () => {
+  return (
+    <LayoutDefault>
+      <section className="flex h-full items-center justify-center">
+        <div className="flex h-[194px] w-[300px] flex-col justify-center gap-1 rounded-xl border-2 border-black bg-white px-[24px] py-[48px] text-center font-new-hero text-lg font-semibold sm:w-[400px] md:w-[581px] ">
+          <div>
+            <p>Thank you!</p>
+            <p>Your submission has been received!</p>
+          </div>
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/screens/RoiCalculatorThankYou.tsx:11]: This `div` can probably be removed

File: .vscode/settings.json
Comment (Mat0ias) [reply .vscode/settings.json:1]: no it's not. Done

File: apps/eo_web/src/router/routes.tsx (hunk 1 above)
Comment (Mat0ias) [reply apps/eo_web/src/router/routes.tsx:25]: Done

File: packages/ui/tailwind.config.cjs (hunk 2 above)
Comment (Mat0ias) [reply packages/ui/tailwind.config.cjs:10]: Done

File: apps/eo_web/src/screens/RoiCalculatorThankYou.tsx (hunk 3 above)
Comment (Mat0ias) [reply apps/eo_web/src/screens/RoiCalculatorThankYou.tsx:11]: Done

**Review by sgoycoechea-lightit — COMMENTED** [review]
Remember to run `pnpm build` and upload the result

**Review by sgoycoechea-lightit — APPROVED** [review]
